docs/rename.md

```diff
@@ -51,6 +58,13 @@ Old IRI,New IRI
 obo:BFO_0000051,fb:BFO_1234567
 ```
 
+If you also want to update the `rdfs:label` of the term you replaced, you can add a third column with the new label value:
```

Comment:
Perhaps add a sentence what happens to the original label (I assume _all_ existing rdfs:label annotations on the term get deleted) and potential annotations on it (language tags, annotation assertions). Also add a note about _multiple labels_, in case someone curates rdfs:label in multiple languages.